### PR TITLE
Added travis-ci job

### PR DESCRIPTION
**Pull request by travi**
You should just need to sign into travis and enable builds for this repo. It fails in this branch, but that is because it needs #2 to be merged in as well.

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+iojs-v2.5
```
Comment (travi) [inline .nvmrc]: I originally tried to bump it up to node 4.0, but there are some dependency problems that I'll let you sort out.

Comment (travi) [inline .nvmrc]: This tells travis which version of node to build against. It's also helpful for anyone that pulls it down and is using nvm as it tells them which version will work too.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+language: node_js
+
```
Comment (travi) [inline .travis.yml]: By default, travis will run the `test` script from the `package.json`

**Comment by dlogankeenan** [comment]
Thanks for doing this Travi!

**Comment by travi** [comment]
Sure thing. Wasn't sure if you'd done anything with travis yet, so thought this would be a good way to introduce you to how simple it is.

**Pull request merged**